test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -870,7 +891,7 @@ public void GenerateSchema_GeneratesSchema_IfParameterHasTypeConstraints()
             var serializerOptions = new JsonSerializerOptions();
             configureSerializer?.Invoke(serializerOptions);
 
-            return new SchemaGenerator(generatorOptions, new JsonSerializerDataContractResolver(serializerOptions));
+            return new SchemaGenerator(generatorOptions, new JsonSerializerDataContractResolver(serializerOptions), Options.Create<MvcOptions>(new MvcOptions()));
```

Comment:
There doesn't appear to be any test cases for when `MvcOptions.SuppressImplicitRequiredAttributeForNonNullableReferenceTypes = true`.